app.js

```diff
@@ -0,0 +1,76 @@
+  alert('Let\'s do this!');
+} else if (userAck.toLowerCase() === 'no' || userAck.toLowerCase () === 'n') {
+  alert('Okay, fine, be that way...but I\'m still going to bother you with questions.');
+} else if (userAck !==true){
+  alert('Okaaaaay, let\'s move on');
+}
+var playerReady = confirm ('Are you ready to play "The Guessing Game?"');
+  if (playerReady == true) {
+  alert('Okay! Let\'s Go!')
+} else {
+  alert('Okay, you\'re starting to piss me off. You can just read about me then. Have fun with that! Okay, just joking, let\'s move on');
+}
+var userAge = prompt ('How old do you think I am?');
+  if (userAge > 33) {
+  alert('Not sure if I should take that as a compliment or...')
+  } else {
+  alert('Good guess! But I\'m a little older.');
+}
+var myHairColor = prompt ('Is my hair brown?');
+if (myHairColor.toLowerCase() === 'yes' || myHairColor.toLowerCase() === 'y') {
+alert('Great job, ' +userName);
+} else if (myHairColor.toLowerCase() === 'no' || myHairColor.toLowerCase () === 'n') {
+alert('You are wrong, ' + userName + '!');
+} else if (myHairColor !==true){
+alert('It was a yes or no question, ' + userName);
+}
+var myEyeColor = prompt ('Are my eyes brown?');
+if (myEyeColor.toLowerCase() === 'yes' || myEyeColor.toLowerCase() === 'y') {
+alert('I have a feeling we\'re going to be good friends, ' +userName);
+} else if (myEyeColor.toLowerCase() === 'no' || myEyeColor.toLowerCase () === 'n') {
+alert('Wow, wrong, ' + userName);
+} else if (myEyeColor !==true){
+alert('I said it was a yes or no question, ' + userName);
+}
+var myHeight = prompt ('Am I taller than 6 feet?');
+if (myHeight.toLowerCase() === 'yes' || myHeight.toLowerCase() === 'y') {
+alert('Yeah, right, I wish, ' +userName);
+} else if (myHeight.toLowerCase() === 'no' || myHeight.toLowerCase () === 'n') {
+alert('I wish I was a little bit taller, I wish I was a baller...');
+} else if (myHeight !==true){
+alert('Why can\'t you answer with a yes or no?, ' + userName);
+}
+var myFacialHair = prompt ('Do I have facial hair?');
+if (myHeight.toLowerCase() === 'yes' || myHeight.toLowerCase() === 'y') {
+alert('You are good at this, ' +userName);
+} else if (myHeight.toLowerCase() === 'no' || myHeight.toLowerCase () === 'n') {
+alert('I\'m trying to grow an impressive beard, you couldn\'t tell? Wrong, just wrong, ' + userName);
+} else if (myHeight !==true){
+alert('Seriously? Yes or no questions, ' + userName);
+}
+var myShirt = prompt ('Final question: Did I wear a blue shirt on Monday?');
+if (myHeight.toLowerCase() === 'yes' || myHeight.toLowerCase() === 'y') {
+alert('Good guess! You\'re right! ' +userName);
+} else if (myHeight.toLowerCase() === 'no' || myHeight.toLowerCase () === 'n') {
+alert('You\'re right! Sike!');
+} else if (myHeight !==true){
+alert('Okay, I\'m done with you, ' + userName);
+}
+confirm('Thanks for playing! Now you can read a little bit more about me.');
+
+console.log(userName + ' answered "What should I call you?" with ' + userName);
```

Comment:
consider grouping these in your code with the prompts and alerts they are linked to so you have access to this information when it is being collected in the flow of your code instead of after the code finishes running.  the logs are for you as the developer, and targeting the state of your code at a certain line will help you debug later.